test/PushComm/unit_tests/SendingNotifications/SendNotifs.t.sol

```diff
@@ -0,0 +1,71 @@
+pragma solidity ^0.8.0;
+
+import { BaseTest } from "../../../BaseTest.t.sol";
+import { CoreTypes } from "../../../../contracts/libraries/DataTypes.sol";
+
+// import "forge-std/console.sol";
+contract SendNotifs_Test is BaseTest {
+    bytes constant _testChannelIdentity = bytes("test-channel-hello-world");
```

Comment:
this variable is used in all 3 files of comm tests, and in BasePushCoreTest.t.sol
If it's getting used in whole test codebase, then it's better to put it in BaseTest itself